Gems/AtomLyIntegration/CommonFeatures/Code/Source/ReflectionProbe/EditorReflectionProbeComponent.h

```diff
@@ -77,6 +78,7 @@ namespace AZ
             AZStd::string m_bakedCubeMapRelativePath;
             Data::Asset<RPI::StreamingImageAsset> m_bakedCubeMapAsset;
             Data::Asset<RPI::StreamingImageAsset> m_authoredCubeMapAsset;
+            float m_bakeExposure = 0.0f;
```

Comment:
Why does EditorReflectionProbeComponent need m_bakeExposure when the ReflectionProbeComponentConfig already has it?

Reply:
It's due to the way I had to structure the UI controls, which required it to be a member of the EditorReflectionProbeComponent.

Reply:
What organization were you having problems with the variable in the component configuration? 
The layout and order of everything in the edit context is oddly determined by the order of fields in the serialize context.
If the content or layout of the properties in the inspector needed to deviate from the component configuration edit context that requires special configuration or additional data in the editor component.